lighthouse-plugin-publisher-ads/audits/gpt-bids-parallel.js

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
+const {auditNotApplicable} = require('../utils/builder');
+const {Audit} = require('lighthouse');
+const {getCriticalGraph} = require('../utils/graph');
+const {getTimingsByRecord} = require('../utils/network-timing');
+const {isImplTag, isBidRequest, getAbbreviatedUrl, getHeaderBidder} = require('../utils/resource-classification');
+const {NOT_APPLICABLE} = require('../messages/messages');
+
+/** @typedef {LH.Artifacts.NetworkRequest} NetworkRequest */
+/** @typedef {LH.Gatherer.Simulation.NodeTiming} NodeTiming */
+
+const id = 'gpt-bids-parallel';
+const UIStrings = {
+  title: 'GPT and bids loaded in parallel',
+  failureTitle: 'Load GPT and bids in parallel',
+  description: 'To optimize ad loading, bid requests should not wait on GPT to load. This issue can often be fixed by making sure that bid requests do not wait on `googletag.pubadsReady` or `googletag.cmd.push`.',
+  headings: {
```

Comment:
Can you update these to match the i18n changes so they're all at the same level. Ex: columnBidder